deploy/lint

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash -e
 
-BASE_DIR="$(dirname "$(dirname "$(readlink -e "$0")")")" # /esp
+# http://stackoverflow.com/a/4774063
+pushd `dirname $0` > /dev/null
```

Comment:
I think you want quotes around the backticks here and on the following line.  (Although if anybody has spaces in their dev server path, they're probably already screwed.)